__tests__/integration/utils/test-utils-integration.js

```diff
@@ -698,4 +698,87 @@ export class TestUtils {
 
     return response.body;
   }
+
+  /**
+   * Get the number of blocks confirming a transaction
+   * @param {string} walletId
+   * @param {string} txId
+   * @returns {Promise<number>}
+   */
+  static async getTransactionConfirmationNumber(walletId, txId) {
+    const response = await TestUtils.request
+      .get('/wallet/tx-confirmation-blocks')
+      .query({ id: txId })
+      .set(TestUtils.generateHeader(walletId));
+
+    if (!response.body.success) {
+      throw new Error(response.text);
+    }
+    return response.body.confirmationNumber;
+  }
+
+  /**
+   * Get the transaction data from the full node
+   * @param {string} txId
+   * @returns {Promise<*>}
+   */
+  static async getFullNodeTransactionData(txId) {
+    // Disabling this eslint rule because of the way API call is done in the lib
+    // otherwise the code would need to be more complex
+    // We should change this when we refactor the way we call APIs in the lib
+    // (this comment also applies for the getMiningInfo call)
+    // eslint-disable-next-line no-promise-executor-return
+    const response = await new Promise(resolve => txApi.getTransaction(txId, resolve));
+
+    if (!response.success) {
+      throw new Error('Failed to get transaction data from full node.');
+    }
+
+    return response;
+  }
+
+  /**
+   * Get the current network height from the full node
+   * @returns {Promise<number>}
+   */
+  static async getFullNodeNetworkHeight() {
+    // Disabling this eslint rule because of the way API call is done in the lib
+    // otherwise the code would need to be more complex
+    // We should change this when we refactor the way we call APIs in the lib
+    // (this comment also applies for the getMiningInfo call)
+    // eslint-disable-next-line no-promise-executor-return
+    const response = await new Promise(resolve => walletApi.getMiningInfo(resolve));
+
+    if (!response.success) {
+      throw new Error('Failed to get network height from full node.');
+    }
+
+    return response.blocks;
+  }
+
+  /**
+   * Wait until a new block is mined in the best chain. Returns the new block height.
+   *
+   * @param {number} currentHeight height to be used as base to wait for a new block
```

Comment:
```suggestion
   * @param {number} [currentHeight=null] height to be used as base to wait for a new block
```
Added the JSDocs parameter syntax as suggestion here.

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-headless/pull/231/commits/ae3405f3cca52ec5720f4449eb6d1f143688345b